ocs_ci/ocs/cluster.py

```
@@ -2391,6 +2391,140 @@ def is_lso_cluster():
+        AssertionError: If pool is expected to exist but is not Ready (critical failure path)
+        TimeoutExpiredError: If pool does not become Ready within timeout after patching
+    """
+    from ocs_ci.ocs.ocp import OCP
+
+    configured_name = get_ec_metadata_pool_name()
+    pool_ocp = OCP(
+        kind=constants.CEPHBLOCKPOOL,
+        namespace=config.ENV_DATA["cluster_namespace"],
+    )
+
+    def pool_is_ready():
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `pool_is_ready`

(ANN202)